### DISCUSSION TITLE
Ping on the pivpn server does not work , cannot connect any devices to the server once I'm connected to wireguard vpn

### DISCUSSION THREAD

**Discussion by Janks14** [pivpn/pivpn#1854] [Q&A]
So I set everything up according to a YouTube tutorial. The pivpn was set , my Huawei router does not have port forwarding settings but a virtual server settings and I put the port forwarding number and IP address of the server in that settings. 

When I'm still connected to the home WiFi network it pings the server but as soon as I connect to the mobile network and connect the wireguard VPN it does not ping the pivpn server

Please assist me with this matter as i have no experience in this but decided to try to do this project

**Comment by orazioedoardo** [comment]
Post the output of `pivpn -d` and try the troubleshooting: https://docs.pivpn.io/faq/#how-do-i-troubleshoot-connection-issues
